src/pages/profile.js

```diff
@@ -23,10 +23,18 @@ import FacebookLogo from '../static/img/facebook.svg';
 
 const ocWebsiteUrl = process.env.WEBSITE_URL || 'https://opencollective.com';
 
-const getProfileData = (id, accessToken) =>
-  process.env.IS_CLIENT
-    ? fetchJson(`/data/getProfileData?id=${id}`)
-    : import('../lib/data').then(m => m.getProfileData(id, accessToken));
+const getProfileData = (id, accessToken, excludedRepos) => {
```

Comment:
Why not have `(id, accessToken, { excludedRepos } )` instead of `(id, accessToken, excludedRepos)`